Intermittent test failure in V5 API

Just caught this failing in V5 API mutation tests, re running passed, probably just not specifying the factory setup with all fields that need to be set for the test to consistently pass:

```

self = <experimenter.experiments.tests.api.v5.test_mutation.TestMutations testMethod=test_update_experiment_probe_sets>

def test_update_experiment_probe_sets(self):
 user_email = "user@example.com"
 probe_sets = [NimbusProbeSetFactory() for i in range(3)]
 experiment = NimbusExperimentFactory.create(
 status=NimbusExperiment.Status.DRAFT, probe_sets=[]
 )
 response = self.query(
 UPDATE_EXPERIMENT_PROBESETS_MUTATION,
 variables={
 "input": {
 "nimbusExperimentId": experiment.id,
 "clientMutationId": "randomid",
 "primaryProbeSetIds": [p.id for p in probe_sets[:2]],
 "secondaryProbeSetIds": [p.id for p in probe_sets[2:]],
 }
 },
 headers={settings.OPENIDC_EMAIL_HEADER: user_email},
 )
 self.assertEqual(response.status_code, 200)
 content = json.loads(response.content)
 result = content["data"]["updateExperimentProbeSets"]
 self.assertEqual(
 result["nimbusExperiment"],
 {
 "id": str(experiment.id),
 "primaryProbeSets": [
 {"name": probe_set.name, "id": str(probe_set.id)}
 for probe_set in probe_sets[:2]
 ],
 "secondaryProbeSets": [
 {"name": probe_set.name, "id": str(probe_set.id)}
 for probe_set in probe_sets[2:]
 ],
 },
 )
 experiment = NimbusExperiment.objects.get(id=experiment.id)
> self.assertEqual(list(experiment.probe_sets.all()), probe_sets)
E AssertionError: Lists differ: [<NimbusProbeSet: Expanded eco-centric orchestration>, <Nim[93 chars]ort>] != [<NimbusProbeSet: Seamless user-facing projection>, <Nimbus[93 chars]ion>]
E 
E First differing element 0:
E <NimbusProbeSet: Expanded eco-centric orchestration>
E <NimbusProbeSet: Seamless user-facing projection>
E 
E - [<NimbusProbeSet: Expanded eco-centric orchestration>,
E - <NimbusProbeSet: Seamless user-facing projection>,
E ? ^
E 
E + [<NimbusProbeSet: Seamless user-facing projection>,
E ? ^
E 
E - <NimbusProbeSet: Configurable value-added support>]
E ? ^
E 
E + <NimbusProbeSet: Configurable value-added support>,
E ? ^
E 
E + <NimbusProbeSet: Expanded eco-centric orchestration>]

experimenter/experiments/tests/api/v5/test_mutation.py:363: AssertionError

```

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/EXP-544)
┆Issue Number: EXP-544

--- a/app/experimenter/experiments/tests/api/v5/test_mutation.py
+++ b/app/experimenter/experiments/tests/api/v5/test_mutation.py
@@ -360,7 +360,8 @@ class TestMutations(GraphQLTestCase):
             },
         )
         experiment = NimbusExperiment.objects.get(id=experiment.id)
-        self.assertEqual(list(experiment.probe_sets.all()), probe_sets)
+        self.assertEqual(set(experiment.probe_sets.all()), set(probe_sets))
+        # These must be compared as lists to ensure ordering is retained.
         self.assertEqual(list(experiment.primary_probe_sets), probe_sets[:2])
         self.assertEqual(list(experiment.secondary_probe_sets), probe_sets[2:])
 

